src/cephadm/cephadm.py

```diff
@@ -8218,6 +8218,8 @@ def disable_systemd_service(unit_name: str) -> None:
             for fname in glob(f'{ctx.log_dir}/cephadm.log*'):
                 os.remove(fname)
 
+        unlink_file(Path('/etc/ceph/podman-auth.json'), missing_ok=True, ignore_errors=True)
```

Comment:
`cephadm.py:8221: error: Name "unlink_file" is not defined`

will need to rewrite this bit

Reply:
Roughly took what the `unlink_file` function was doing and wrote it here. Not ideal, but we aren't going to end up backporting that change to reef, so this is probably our best option.